Documentation/Getting-Started/Configuration-Guide.md

```diff
@@ -1063,6 +1063,19 @@ seconds. The interval is incremented until the value of `max_retry_interval` is
 reached at which point the listener attempts to bind to the interface every
 `max_retry_interval` seconds.
 
+#### `session_track_trx_state`
+
+Enable or disable getting session transation state via session track mechanism. This
+parameter takes a boolean value. Default False.
+
+Get current session transaction state from server side will be more accurate.
```

Comment:
I would add a comment here that this is also faster as no parsing is needed on MaxScale. Also documenting that this will correctly track transaction state inside stored procedures would be good.